### ISSUE TITLE
harvesting fully grown crops reduces rep by lots yet planting new crops increases it but only by a little bit

### ISSUE THREAD

**Issue by Robomnz** [ToroCraft/ToroQuest#210]
title explains it i built a farm in a town because the town didnt have any and after a while i noticed i had dropped into neg rep when the only thing i was doing was farming in my own farm that i had built... maybe make it so if the town discovers that you are the player that hoed the ground and planted/harvest crops it gives rep because i noticed that when harvesting crops it decreases rep by lots and thats a really sad discovery after working so hard on a farm so i could get rep in the first place and also i made that farm if anything you would expect the town to be happy with you for doing that but nope apparently not..

**Comment by Robomnz** [comment]
this means you loose rep if you make a farm in a town and you harvest crops that you replant right after maybe instead make it so you only loose rep if you dont replant the crops you harvest after lets say a minute

**Comment by Robomnz** [comment]
and replanting the crops increases rep ofc if anything both harvesting and replanting the crops should increas rep as you are doing the township a favor by managing the farms

**Comment by zeriley** [comment]
The idea is that when you harvest the crop that was within the civilization, it looks like you're stealing it.  And if you never replant, then you are stealing from them.  There is an equally random chance on harvesting and planting of losing or gaining rep, and since you usually get multiple plantable drops from each harvest, if you are truly managing the farm, the rep should go up considerably over time.  Ideally, we would know if you replanted, but with the way the event system is, it's not easy to do.

I know when I played with ToroQuest, I exploited the farming to get an incredible amount of reputation in a civilization.
